packages/synapse-react-client/src/theme/palette/Palettes.ts

```diff
@@ -78,7 +78,8 @@ export const palette: PaletteOptions = {
   },
   // The neutral palette can be used to color components like buttons in grayscale
   neutral: generatePalette('#878e95'), // grey-700
-  darkPrimary: generatePalette('#0b1218'), // primary-900
+  darkPrimary: generatePalette('#164B6E'),
+  lightPrimary: { ...generatePalette('#f8f9fa'), contrastText: '#164B6E' },
```

Comment:
`darkPrimary` was only used on the SynapseHomepage, so update to reflect the current design. Create a similar `lightPrimary` color that can also be used on the homepage.